components/blocks/nav/SiteNavbar/SiteNavbar.test.ts

```diff
@@ -0,0 +1,94 @@
+import { ref } from 'vue'
+import { useState } from '#app'
+import SiteNavbar from './SiteNavbar.vue'
+import * as apiComposables from '@/composables/api'
+import { User } from '@/lib/api/data-contracts'
+import { fireEvent, stubbedRender } from '@/testUtils'
+
+vi.mock('#app', () => ({
+  useState: vi.fn(),
+}))
+
+afterEach(() => {
+  vi.restoreAllMocks()
+  vi.clearAllMocks()
+})
+
+describe('<SiteNavbar />', () => {
+  it('should render without crashing', () => {
+    useState.mockImplementation(() => ref())
```

Comment:
Why the mock to call `ref` here? Does the test error without it (assuming so)?

Reply:
Oh, I missed that my initial mocking of it, just sets it as `vi.fn()`. I should be able to set it there instead, and then this won't be needed